README.md

```diff
@@ -80,21 +81,20 @@ make
 sudo make install
 ```
 
-On macOS, it is easiest to install Z3 from Homebrew. If you do wish to install
-from source, make sure to install it to an appropriate prefix (e.g. `/usr/local`
-on Intel machines).
+On macOS, it is easiest to install Z3 from Homebrew.
+If you wish to install from the source, install it to an appropriate prefix (e.g. `/usr/local` on Intel machines).
 
 #### Ubuntu
 
 On Ubuntu >= 18.04 (for example):
 
 ```sh
 sudo apt-get install --yes                                                             \
-            autoconf bison clang-10 cmake curl flex gcc jq libboost-test-dev           \
+            autoconf bison clang-12 cmake curl flex gcc jq libboost-test-dev           \
             libcrypto++-dev libffi-dev libgflags-dev libjemalloc-dev libmpfr-dev       \
-            libprocps-dev libsecp256k1-dev libssl-dev libtool libyaml-dev lld-10       \
-            llvm-10-tools make maven netcat-openbsd openjdk-11-jdk pkg-config          \
-            protobuf-compiler python3 python3-dev python3-pip rapidjson-dev time      \
+            libprocps-dev libsecp256k1-dev libssl-dev libtool libyaml-dev lld-12       \
+            llvm-12-tools make maven netcat-openbsd openjdk-11-jdk pkg-config          \
+            protobuf-compiler python3 python3-dev python3-pip rapidjson-dev time       \
             zlib1g-dev
 pip3 install poetry
 ```
```

Comment:
I believe we require at least Ubuntu 22 at this point? https://github.com/runtimeverification/evm-semantics/issues/1536

Reply:
And ubuntu 22 comes with a newer LLVM requirement, I think llvm-14 if I'm not mistaken: https://github.com/runtimeverification/k/blob/master/.github/workflows/test-pr.yml#L92

Reply:
I will update the Ubuntu version. I'm currently running ubuntu 22.04 with `llvm-12`.